### PR TITLE
[PE-3971] Added meta description to docs to improve SEO rating

### PR DESCRIPTION
**Pull request by janibolkvadze**
Added support of defining unique description meta tags for docs pages

@mattnickles @jllr 

https://ctl.atlassian.net/browse/PE-3971

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/462/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/463/ for more details.

File: src/website/layouts/default.pug [hunk 1]
```diff
@@ -1,11 +1,19 @@
+- var defaultMetaDescription = "Chi (pronounced chee) provides developers with a collection of HTML and CSS patterns to build fast, reusable, and consistent responsive interfaces."
```
Comment (mattnickles) [inline src/website/layouts/default.pug]:
```suggestion
- var defaultMetaDescription = "Chi provides developers with a collection of HTML and CSS patterns to build fast, reusable, and consistent responsive interfaces."
```

File: src/website/views/getting-started/index.pug [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Introduction
 order: 1
+description: Chi design system of CenturyLink provides developers with a collection of HTML and CSS patterns to build fast, reusable, and consistent responsive interfaces.
```
Comment (mattnickles) [inline src/website/views/getting-started/index.pug]:
```suggestion
description: Chi provides developers with a collection of HTML and CSS patterns to build fast, reusable, and consistent responsive interfaces.
```

**Comment by mattnickles** [comment]
@janibolkvadze

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/470/ for more details.

**Pull request merged**